aiogram/types/input_file.py

```diff
@@ -68,13 +76,15 @@ def from_file(
         """
         if filename is None:
             filename = os.path.basename(path)
-        with open(path, "rb") as f:
-            data = f.read()
-        return cls(data, filename=filename, chunk_size=chunk_size)
+        buffer = io.BytesIO()
+        with open(path, "rb") as fp:
+            while chunk := fp.read(chunk_size):
+                buffer.write(chunk)
+        return cls(buffer, filename=filename, chunk_size=chunk_size)
```

Comment:
This modification didn't change anything because `from_file` is still blocking.

Instead of this change, I see the sense of adding another method called `load_from_file`, which should be asynchronous and use `aiofiles` to read the contents into the bytes variable.

Note that both methods should be here because you can initialize `InputFile` globally to reuse a static file without performing disk I/O operations.